bpf/aya-bpf/src/programs/sk_buff.rs

```diff
@@ -65,6 +66,79 @@ impl SkBuffContext {
         }
     }
 
+    /// Writes bytes from the SKB context to the given bytes slice.
+    ///
+    /// Reads at most `self.len() - offset` bytes from the SKB context.
+    ///
+    /// # Examples
+    ///
+    /// With a `PerCpuArray` (with size suitable for high jumbo frames MTU):
```

Comment:
```Read into a `PerCpuArray`.```

I wouldn't mention jumbo frames and would make the buffer smaller. It doesn't add to the example and it will be unfamiliar terminology to newcomers.